apache/superset
# other: Add Helm chart option for configOverrides through file

### SUMMARY
To extend `superset_config.py`, the Superset Helm chart currently allows users to [add code](https://github.com/apache/superset/blob/master/helm/superset/values.yaml#L113) under `configOverrides` in `values.yaml`. However, it would be better if this code was in a separate `.py` file in the chart directory. This PR enables that through a similar setup. 

### TESTING INSTRUCTIONS

Create a `test.py` in `helm/superset` and add `test: test.py` as an entry under `configOverridesFiles`. Install with
```
helm install superset helm/superset
```
After installing,
```
k get secret/superset-config -o json | jq '.data."superset_config.py"' -r | base64 -D
```
should show the code in test.py
